superset-frontend/src/CRUD/CollectionTable.tsx

```diff
@@ -292,7 +292,7 @@ export default class CRUDCollection extends React.PureComponent<
             </Button>
           )}
         </span>
-        <table className="table">
+        <table data-test="table" className="table">
```

Comment:
maybe `crud-table` just to make sure we don't find ourselves running into an issue with two tables conflicting on a page? Sorry, I know these problems all seem hypothetical, but this is an excuse to _not_ run into the same issues we do with class names!